proposals/lip-0044.md

```diff
@@ -99,10 +123,10 @@ validatorAccountSchema = {
 
 ##### Properties and Default values
 
-The validator account holds the generator and BLS keys of a registered validator. In this section, we describe the properties of a validator account. These properties are set by the `registerValidatorKeys` function, called for instance during the processing of the [delegate registration command][lip-0057] or the [authority registration command][lip-0047].
+The validator account holds the generator and BLS keys of a registered validator. In this section, we describe the properties of a validator account. These properties are set by the [`registerValidatorKeys` function](#registerValidatorKeys), called for instance during the processing of the [delegate registration command][lip-0057] or the [authority registration command][lip-0047].
```

Comment:
improve link to registarValidatorKeys:   [registerValidatorKeys](#registerValidatorKeys) function

Reply:
The link is already present, not sure what you mean here.